### ISSUE TITLE
Update server reflection tutorial

### ISSUE THREAD

**Issue by dfawley** [grpc/grpc-go#4593]
https://github.com/grpc/grpc-go/blob/master/Documentation/server-reflection-tutorial.md#enable-server-reflection

Does not specify the `-plaintext` flag is needed.  Also consider updating to use `grpcurl` instead of `grpc_cli`.

**Comment by easwars** [comment]
@buzzsurfr : Is this something you would be interested in taking on, once https://github.com/grpc/grpc-go/issues/5684 is done?

**Comment by buzzsurfr** [comment]
Sure thing.

**Comment by easwars** [comment]
Thank you very much !!

**Comment by buzzsurfr** [comment]
If we update to use `grpcurl`, then the `--channel_creds_type` flag is no longer relevant. Should we include instructions for both or just move to `grpcurl`?

**Comment by easwars** [comment]
I think moving to `grpcurl` should be good enough.
